### DISCUSSION TITLE
Possible to add custom chain for IOS version?

### DISCUSSION THREAD

**Discussion by yulafezmesi** [trustwallet/wallet-core#3818] [Q&A] [answered]
The adding custom chain is supported for Android however can't seen on IOS. the option available on IOS?

https://help.avocado.instadapp.io/en/articles/7037927-using-trustwallet-android-to-use-avocado

**Comment by rkokhatskyi** [comment, marked as answer]
Hi @yulafezmesi, the iOS app does not currently support custom chains, however this feature is in our roadmap for future updates.

---
Your question is not related to the wallet core repository, for additional support or any other inquiries about Trust Wallet, please reach out to our Customer Support team by creating a ticket here: https://support.trustwallet.com/en/support/tickets/new

**Reply by yulafezmesi** [reply]
gotcha, thanks @rkokhatskyi!

also wondering, can we add/possible Avocado Chain by default? Avocado is smart contract wallet, not actual EVM network.

**Reply by satoshiotomakan** [reply]
Hi @yulafezmesi, please submit a ticket via https://support.trustwallet.com/en/support/tickets/new or https://governance.trustwallet.com
